build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,167 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$OVSVersion-win64.zip"
+$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$OVSDownloadDir = $WorkDir
+$OVSInstallDir = "C:\openvswitch"
+$OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
+    if (Test-Path -Path $OVSInstallDir) {
+        Write-Host "$OVSInstallDir already exists, exit OVS installation."
+        exit 1
```

Comment:
Incase when Pod restarts and OVS is already installed, it should just skip OVS download and installation steps and jump to ConfigOVS. But here it returns `exit 1` which will stop the execution of this script.